pkg/filehandler/filehandler.go

```diff
@@ -38,21 +40,29 @@ func GzipFileReader(gzFilePath string) *gzip.Reader {
 		log.Fatal(err)
 	}
 
-	gzipReader.Close()
+	err = gzipReader.Close()
```

Comment:
Will the contents of gzipReader still be  accessible if it's closed here?

Reply:
Yes. The only thing is in case of error closing the handler maybe we can change it to just inform but no exit the app. What do you think?

Reply:
Yes, I think that  it makes sense, if there's a problem reading one file, but there are other files to read, there is no need to crash.

Reply:
You're right. I changed it. Thank you Joan.